src/pages/ProfilePage.tsx

```diff
@@ -179,60 +269,93 @@ export default function ProfilePage() {
           {/* Username */}
           <div style={{ padding: '5px' }}>
             <hr style={{ borderTop: '1px' }} />
-            <Grid container gap={2} alignItems="center">
-              <Grid item xs={3.5}>
+            <Grid container columnSpacing={2} alignItems="center">
```

Comment:
not sure how the grids are set up here, but I believe for every grid container, its grid items are supposed to add up to 12